knowledge-base/how-to-show-the-page-scroll-mode-button.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Showing the Page Scroll Mode Button
+page_title: Showing the Scroll Mode Button of the Page
+description: "Learn how to show the Scroll Mode button on the page when using Telerik Reporting in web applications."
+slug: telerikreporting/using-reports-in-applications/display-reports-in-applications/web-application/html5-report-viewer/customizing/how-to-show-the-page-scroll-mode-button
+tags: telerik, reporting, showing, the, page, scroll, mode, button
+previous_url: /using-reports-in-applications/display-reports-in-applications/web-application/html5-report-viewer/customizing/how-to-show-the-page-scroll-mode-button
+published: True
+type: how-to
+res_type: kb
+---
+
+## Environment
+
+<table>
+	<tbody>
+		<tr>
+			<td>Product</td>
+			<td>Progress® Telerik® Reporting</td>
+		</tr>
+		<tr>
+			<td>Version</td>
+			<td>Q2 2012 and newer</td>
+		</tr>
+	        <tr>
+			<td>Report Viewers</td>
+			<td>WinForms, WPF, ASP.NET</td>
+		</tr>
+	</tbody>
+</table>
+
+## Description
+
+How can I show the **Scroll** mode button of the page in the HTML5 Viewer toolbar area?
+
+## Solution
+
+To achieve the required scenario, accomplish the following steps. Note that you have to adapt all path references in the article according to your project setup. For more information, refer to the [ASP.NET Web Project Paths](http://msdn.microsoft.com/en-us/library/ms178116.aspx) MSDN article. The mentioned Report Viewer Template files are provided with your [Telerik Reporting Installation]({%slug telerikreporting/installation%}#directories-and-asemblies).
+
+1. Find the Telerik Report Viewer template in your installation folder located in `%programfiles(x86)%\Progress\Telerik Reporting`.
```

Comment:
the folder is like:
"C:\Program Files (x86)\Progress\Telerik Reporting R1 2022\Html5\ReportViewer\templates"